base/shell/explorer/traywnd.cpp

```diff
@@ -2224,6 +2461,9 @@ class CTrayWindow :
         /* Create the Start button */
         m_StartButton.Create(m_hWnd);
 
+        /* Create the 'Show Desktop' button */
+        m_ShowDesktopButton.DoCreate(m_hWnd, IsShowDesktopButtonNeeded());
```

Comment:
I was under the impression that the feature was meant (in case it's not enabled) to not create the button at all, i.e. _*disable*_ completely the feature.
Here you always create it, but just show or hide it, meaning that one could still send messages to its window (programmatically, thus still being able to use the feature) if one knows it exists there.

Reply:
OK. Fixed.